packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -627,6 +640,29 @@ export class RestApi extends RestApiBase {
    */
   public _attachMethod(method: Method) {
     this.methods.push(method);
+
+    // add this method as a dependency to all deployments defined for this api
+    // when additional deployments are added, _attachDeployment is called and
+    // this method will be added there.
```

Comment:
Please copy over the comment from the original code that contains the link and quote from CF docs explaining why this dependency is needed. 
Helps posterity without having to fish for the commit description.

Reply:
ok, no problem